File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Prompt a user to input height with do... while... loop
    int x;
    do
    {
        x = get_int("Height: ");
    }
    //Use operators to set the range
    while (x < 1 || x > 8);

    //Create nested loops and conditional statements to make dots and hashes
    for (int i = 0; i < x; i++)
    {
        for (int j = 0; j < x; j++)
        {
            if (j < x - i - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: mario.py
# create while loop with try except statement to catch valueerror
while True:
    try:
        height = int(input("Enter a height: "))
        if 1 <= height <= 8:
            break
    except ValueError:
        pass

# use a nested for-loop to iterate through rows and columns
for i in range(height):
    for j in range(height):
        if j < height - i - 1:
            # use end='' to combine characters in one row
            print(' ', end='')
        else:
            print('#', end='')
    print()
